code/API_definitions/number-verification.yaml

```diff
@@ -2,43 +2,67 @@ openapi: 3.0.3
+    This API enables an API Consumer to verify or retrieve the phone number of the mobile device being used to access their service
+
+    # The Authentication Request
+
+    ## Authentication Request with a temporary token
+
+    If the API Consumer has a TS.43 temporary token created on the mobile device then this API works over all connections e.g. WiFi taking advantage of the SIM-Based authentication.
+    How the API Consumer gets a TS.43 temporary token is out-of-scope of the API definition.
+    The API Consumer sends the temporary token to their backend and which sends a CIBA Authentication Request, as described in the current release [CAMARA APIs Access and User Consent Management](https://github.com/camaraproject/IdentityAndConsentManagement), with a parameter "login_hint=operatortoken:<temporary token>".
+
+    ## Authentication Request without a temporary token
+
+    If the API Consumer does not have a TS.43 temporary token then the API Consumer must use OpenId Connect Authorization Code Flow as described in the current release of [CAMARA APIs Access and User Consent Management](https://github.com/camaraproject/IdentityAndConsentManagement).
+    This mechanism of utilizing the mobile connection does not work on a fixed-net Internet connection nor over WiFi or VPN.
+    The API Consumer should not send a login_hint in the Authentication Request.
 
     # Resources and Operations overview
-    This API currently provides two endpoints which both require a **3-legged token** and authentication via **mobile network** (**excluding** for example by SMS/OTP or user/password as an authentication method):
-    - The first one checks if the user mobile phone number matches the phone number associated with the mobile device. It can receive either a hashed or a plain text phone number as input and it compares the received input with the authenticated user's phone number associated to the access token in order to respond **true/false**.
-    - The next one retrieves the phone number associated to the user's token and returns it so the verification can be made by the service provider.
+
+    This API currently provides two endpoints which both require a **3-legged token**:
+    - The first one checks if the user's mobile phone number matches the phone number associated with the mobile device.
+      It can receive either a hashed or a plain text phone number as input.
+      It compares the received phone number with the user's phone number associated to the access token in order to respond **true/false**.
+    - The next one retrieves the phone number associated to the access token and returns it.
```

Comment:
```suggestion
    - The /device-phone-number endpoint returns the phone number associated by the network operator with the SIM in the end user's device.
```

Reply:
I like that but that changes much from the original. Also, if we change "the next one" then I suggest to also use the name/path for the first one.

Reply:
Add the path value in both paragraphs?

Reply:
I thought I had proposed changing the first paragraph as well. Maybe I forgot to submit that suggestion. I'll suggest it again.